lib/src/debian.dart

```diff
@@ -0,0 +1,282 @@
+import 'dart:io';
+
+import 'package:cli_pkg/src/standalone.dart';
+import 'package:grinder/grinder.dart';
+import 'package:path/path.dart' as p;
+
+import 'config_variable.dart';
+import 'github.dart';
+import 'info.dart';
+import 'utils.dart';
+
+/// The GitHub repository slug (for example, `username/repo`) of the PPA
+/// repository for this package.
+///
+/// This must be set explicitly.
+final debianRepo = InternalConfigVariable.fn<String>(
+    () => fail("pkg.debianRepo must be set to deploy to PPA repository."));
+
+/// The contents of the debian packages's control file.
+///
+/// By default, it is loaded from `control` at the root of the repository.
+/// It's modifiable.
+///
+/// `cli_pkg` will automatically update the `version` field when building the package.
+final controlData = InternalConfigVariable.fn<String>(() =>
+    File("control").existsSync()
+        ? File("control").readAsStringSync()
+        : fail("pkg.controlData must be set to update Debian package."));
+
+/// The fingerprint for GPG key to use when signing the release.
+///
+/// By default, this comes from the `GPG_FINGERPRINT` environment variable.
+final gpgFingerprint = InternalConfigVariable.fn<String>(() =>
+    Platform.environment["GPG_FINGERPRINT"] ??
+    fail("pkg.gpgPassphrase must be set to deploy to PPA repository."));
```

Comment:
We should be able to determine the fingerprint as the GitHub Action also does that, but I couldn't figure out a way for that. Let me try again.

I read in the manual of GPG command the following and further @stof also suggested using fingerprints over emails.
> The best way to specify a key Id is by using the fingerprint.  This avoids any ambiguities in case that there are duplicated key IDs.

Reply:
I can imagine reasons the end user might want to choose to use an email lookup, but I'm fine not adding that unless someone actively asks for it.

Reply:
Passing `email` in `pkg.gpgFingerprint` will work as well since it only references the key. The command doesn't explicitly specify that it's a fingerprint.

Reply:
Maybe it should just be `pkg.gpgKey` then, to match the CLI parameter name.

Reply:
I guess that's ambiguous with the contents of the private key itself... I guess `pkg.gpgFingerprint` is fine.

Reply:
Sure. We can still mention in the documentation that passing the email here will also work.

Reply:
Regarding determining the fingerprint from the key, I found that passing `--import-options import-show` while importing gives the following output. So we can extract the fingerprint from the second line in the STDOUT.

```
sec#  rsa4096 2022-01-14 [SC]
      FINGERPRINT
uid            NAME <EMAIL>
ssb#  rsa4096 2022-01-14 [E]
```

Reply:
Found a better fit here using `--with-colons` argument. Here the output can be easily parsed and has a format something like this.

```
....other lines....
fpr:::::::::FINGERPRINT:
...other lines....
```